src/main/java/me/dinowernli/grpc/prometheus/Common.java

```diff
@@ -0,0 +1,60 @@
+package me.dinowernli.grpc.prometheus;
+
+import io.grpc.Metadata;
+import io.grpc.Metadata.Key;
+import io.prometheus.client.SimpleCollector;
+
+import java.util.ArrayList;
+import java.util.List;
+
```

Comment:
Can we add a few unit tests for this? I think that will make this logic easier to maintain and extend.

Reply:
Added. Only couldn't add a unit test for addLabels() without either implementing an elaborate counter collector creation and use scenario (which sounds more integration than unit), or adding a new dependency on a mocking library.
I reckon it's not really needed since it's tested through its use in the tested scenarios under integration.